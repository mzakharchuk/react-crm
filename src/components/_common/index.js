export * from './header/Header'
export * from './sidebar/Sidebar'
export * from './input/TextInput'