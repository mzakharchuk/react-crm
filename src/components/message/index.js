export * from './MessageForm'
export * from './MessageList'
export * from './MessagePage'
export * from './TabContent'