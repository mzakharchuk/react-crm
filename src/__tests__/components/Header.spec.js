import React from 'react'
import {Header} from '../../components/_common'
import renderer from 'react-test-renderer'

describe('The header ', () =>{
    it('should renders  snapshot', ()=>{
        const tree =renderer
        .create(<Header/>)
        .toJSON();

        expect(tree).toMatchSnapshot()
    })
})