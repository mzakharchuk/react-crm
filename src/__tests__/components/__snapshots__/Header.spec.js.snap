// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The header  should renders  snapshot 1`] = `
<nav
  className="navigation-bar row"
>
  <div
    className="col-md-10"
  >
    <a
      className="logo col-md-6"
      href="/"
    >
      <i
        className="fab fa-500px fa-2x logo-color"
      />
    </a>
  </div>
  <div
    className="col-md-2 navigation-bar__profile"
  >
    <div>
      <div
        className="dropdown"
      >
        <a
          className="dropbtn"
        >
          Profile
          <i
            className="fa fa-caret-down"
          />
        </a>
        <div
          className="dropdown-content"
        />
      </div>
    </div>
  </div>
</nav>
`;
